src/app/server/RendezvousServer.cpp

```diff
@@ -94,6 +96,9 @@ CHIP_ERROR RendezvousServer::WaitForPairing(const RendezvousParameters & params,
         ReturnErrorOnFailure(GetAdvertisementDelegate()->StartAdvertisement());
     }
 
+    // reset all advertising
+    app::Mdns::StartServer(true, true);
```

Comment:
Why is additionalCommissioning true here?

Reply:
StartServer seems like an overall incorrect name. "StartResponder" is more like it. Furthermore, starting/stopping advertisement is separate from starting/stopping an  mDNS responder.

For instance: Thread nodes neither use mDNS (they use unicast DNS to SRP) nor run a server (they register to SRP on border router). This legacy naming from earlier is starting to make the code much harder to read/understand.

Reply:
added comment to code regarding why these arguments
I prefer not to change the name of the method in this PR though happy to take that in a future one.